Neos.Cache/composer.json

```diff
@@ -14,7 +14,7 @@
     },
     "require-dev": {
         "mikey179/vfsstream": "^1.6.1",
-        "phpunit/phpunit": "~8.1"
+        "phpunit/phpunit": "~8.5"
```

Comment:
Required because of `expectExceptionMessageMatches()` in PersistenceManagerTest - that method was introduced in 8.4 actually, but 8.4 fails with `Cannot instantiate abstract class Neos\Eel\Tests\Unit\AbstractEvaluatorTest` due to the dataProviders there.